elasticsearch-model/gemfiles/3.0.gemfile

```diff
@@ -9,4 +9,4 @@ gemspec path: '../'
 
 gem 'activemodel',  '>= 3.0'
 gem 'activerecord', '~> 3.2'
-gem 'mongoid',      '>= 3.0'
+gem 'mongoid',      '~> 3.0'
```

Comment:
I like this change, since Mongoid 4 will bring a lot of breaking changes. This PR is about bringing AR and Mongoid support closer together, so it makes sense that both are locked into their 3.x releases.